Sources/NIOCrashTester/CrashTests+OnLoopSendable.swift

```diff
@@ -0,0 +1,72 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020-2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import NIOCore
+import NIOPosix
+
```

Comment:
Can you comment these tests out on iOS/watchOS/tvOS please?

Reply:
@Lukasa the other crash tests aren't, also they are only in `NIOCrashTester` which is a binary which isn't compiled on those platforms, right>?

Reply:
See #2369.

Reply:
oh my, let's merge @carolinacass 's PR and I'll fix my merge conflicts with it.

Reply:
@Lukasa actually I copied the merge-conflicting code so maybe an auto-merge will fix it. I think this is done.